### DISCUSSION TITLE
Submitter versus source

### DISCUSSION THREAD

**Discussion by webtrees-pesz** [FamilySearch/GEDCOM#426] [Q&A] [answered]
How is it documented if another researcher provides previously unknown events or facts about a person? 

Strictly speaking, the additional researcher would be the submitter of the data and should be documented as such.
However, the EVENT_DETAIL structure does not have a SUBM tag. 

In this case, should the submitter be documented as the source? If yes, how do you differentiate this source information from other sources provided?

**Comment by dthaler** [comment, marked as answer]
You asked "How is it documented if another researcher provides previously unknown events or facts about a person?"
There are a couple subcases: (a) the researcher provided the information in the form of a GEDCOM file, (b) the researcher provided the information in some other way, such as text in the body of email or a letter, or verbally such as in an interview.  You may know some of this already but I wanted this Q&A to be more comprehensive and helpful to others too.

Let's cover the GEDCOM case first.  The GEDCOM spec defines submitter as:
> The submitter record identifies an individual or organization that contributed information contained in the dataset.

The BCG (Board for Certification of Genealogists) Genealogical Standards Manual (2008), p.3 references Elizabeth Shown Mills for source citation models, which is one reason why the work-in-progress GEDCOM Citations repository (https://github.com/dthaler/gedcom-citations) references Elizabeth Shown Mills' _Evidence Explained_ for sourcing, so it's worth explaining what EE says about citing a contributor of information in GEDCOM files.  EE section 3.44 (pages 156-157) covers this.

Examples in that section include:
> _**Source List Entry**_
> "Kincaid GEDCOM File." Compiled by TexAnna Kincaid [(E-ADDRESS), & MAILING ADDRESS], Boise Idaho. Supplied by Kincaid, 23 August 2006. Copy held by [RESEARCHER & CONTACT INFORMATION].
>
> _**First Reference Note**_
> 2. Biographical sketch of Lois Kincaid, no. 1234, "Kincaid GEDCOM File," compiled by TexAnna Kincaid [(E-ADDRESS), & MAILING ADDRESS], Boise, Idaho; supplied by Kincaid, 23 August 2006, with no citation of source; copy held by [RESEARCHER & CONTACT INFORMATION].
> 3. Biographical sketch of John Kincaid, no. 321, "Kincaid GEDCOM File," by TexAnna Kincaid, citing "Resignations of militia officers, November 1832-January 1833 General Assembly Session, North Carolina State Archives."

(Source List Entry corresponds to `SOURCE_RECORD` and First Reference Note corresponds to `SOURCE_CITATION`, so yes this means the submitter is recorded as a source.)

And Mills gives some additional exposition:
> CITING CONTACT INFORMATION FOR THE COMPILER
> [...]
> when you cite reports and GEDCOM files prepared by private researchers, contact information is essential.
>
> CITING FILE VS. AUTHOR IN SOURCE LIST ENTRY
> [...]
> Examples 2 and 3 cite the file under the file name, with the compiler's name relegated to second place. The difference lies in the fact that contact information needs to be supplied for the compiler of the GEDCOM file.  If her name was the lead element in the citation, then her e-mail address and mailing address would be awkwardly placed in the citation _before_ the name of the file.
>
> CITING GEDCOM FILES
> [...]
> A GEDCOM file that purports to be reliable will cite its sources, in which case you will examine those other sources and cite whatever you examined. If you cite a GEDCOM file temporarily while waiting access to other materials, then you add a notation quoting the cited source or else note that the asserted fact is undocumented.

So, a `SUBM` would be important whenever "contact information is essential", but would not be important once one has examined whatever source the original submitter cited.  However, as you correctly noted, `SUBM` does not appear at the `EVENT_DETAIL` level, only under an `INDI`, `FAM`, or `HEAD`.  As such, you need some way to store the contact information under a `SOURCE_RECORD`.  Without using any extensions, the most natural place is to put contact information in the `PUBL` field of the `SOURCE_RECORD`.

There are some currently proposed extensions available for feedback that may help with this:
* The [Source Derivation](https://github.com/dthaler/gedcom-citations/blob/main/source-derivation-extension.md) can allow expressing the researcher's cited (but not yet examined by you) source separate from citing the researcher, and separate from any other sources that you have examined. The researcher and any sources you examined would be cited under the `EVENT_DETAIL` and any sources cited by the researcher (but not examined by you) could be cited under the `SOURCE_CITATION` using the extension.
* The [Source Templates Extension](https://github.com/dthaler/gedcom-citations/blob/main/templates-extension.md) may allow better ways to store contact information in a `SOURCE_RECORD`.

For the private correspondence case, see EE pages 110, 112, and 113 for quick summaries, with elaboration in sections 3.32, 3.41, 3.42, 3.43.  I won't quote them here except to say that the answer for how to record in GEDCOM turns out, I believe, to be the same as above.

**Reply by dthaler** [reply]
Discussion in GEDCOM Steering Committee 8 FEB 2024:
Some additional notes:
* The SUBM substructure also contains metadata like CHAN, which is not the source or submitter itself, it is an attribution of the user account that made the edit.  
* If such information is desired, one can use an extension such as a relocated standard structure such as `_SUBM` of type `g7:SUBM`, under `<<EVENT_DETAIL>>`.
